index.d.ts

```diff
@@ -339,6 +339,11 @@ export class TimeoutError extends Error {
 	constructor();
 }
 
+/**
+Symbol for aborting retries in the `beforeRetry` hook
```

Comment:
```suggestion
Symbol for aborting retries in the `beforeRetry` hook.
```

Would be useful to describe how it's supposed to be used and an example.

Reply:
Yes, i was going to document its usage as well. Thanks.